### DISCUSSION TITLE
Can we connect to external Spark Service

### DISCUSSION THREAD

**Discussion by singh-b** [feast-dev/feast#1346] [Q&A] [answered]
My organisation already has a big data platform with Spark services running. Theoretically I could connect to existing Spark services for ingestion job, is that correct understanding?

**Comment by woop** [comment, marked as answer]
Hey @singh-b!

What system are you using to manage Spark jobs?

**Reply by singh-b** [reply]
Thanks for quick turnaround @woop 
We use YARN for hadoop infra. I see many large scale companies ( atleast few known to me) have invested heavily in YARN and it would be good to support that, in case it does not work in the current state.

**Reply by woop** [reply]
We don't currently support YARN. We currently support the following Spark job managers https://github.com/feast-dev/feast-spark/tree/master/python/feast_spark/pyspark/launchers

We'd love to have support for YARN, but it's unlikely that we will implement it soon since we already support SparkOp for open source deployments. We'd be happy to receive a PR if you'd like to take a stab at it!
